Fix epub doc configuration

andreas-bok-sociomantic opened the pull request:
Some characters were written as html entity names.
Those characters are &nbsp; and &copy have been
encoded with their equivalent html entity number

Files not relevant for the epub build have been
excluded as well using the 'epub_exclude_files'
setting.

ben-palmer-sociomantic commented:
I would change the first paragraph of the commit message to:
```
Some characters were written as html entity names 
(&nbsp; and &copy). They are now encoded with their 
equivalent html entity number
```

ben-palmer-sociomantic commented on docs/conf.py.in:
```diff
@@ -222,6 +222,8 @@ latex_elements = {
     'classoptions': 'openany,oneside', 'babel' : '\\usepackage[english]{babel}'
 }
 
+epub_exclude_files = ['.doctrees/compiling.doctree', '.doctrees/examples/statement.doctree']
```
I would split these changes in to a separate commit.

andreas-bok-sociomantic committed:
Replace html entity names with their entity number

Some characters were written as html entity names
(&nbsp; and &copy). They are now encoded with their
equivalent html entity number

andreas-bok-sociomantic committed:
Add exclude files for epub docs config

Files not relevant for the epub build have been
excluded as well using the 'epub_exclude_files'
setting.

andreas-bok-sociomantic commented:
- Updated ci message 
- Split into two commits 
- Removed the `html_use_smartypants = True` change since it is `True` by default

ben-palmer-sociomantic commented:
The first commit message could still be updated then LGTM.

andreas-bok-sociomantic commented:
> The first commit message could still be updated then LGTM.

True, lost in rebase :)

andreas-bok-sociomantic commented:
ping
